app/docs/setup_code.tsx

```diff
@@ -310,69 +371,84 @@ export default class SetupCodeComponent extends React.Component {
             for more info.
           </div>
         )}
-        <b>Copy to your .bazelrc</b>
-        <code data-header=".bazelrc">
-          <div className="contents">
-            <div>{this.getResultsUrl()}</div>
-            <div>{this.getEventStream()}</div>
-            {this.state.cacheChecked && <div>{this.getCache()}</div>}
-            {this.state.cacheChecked && <div>{this.getCacheOptions()}</div>}
-            {(this.state.cacheChecked || this.state.executionChecked) && <div>{this.getRemoteOptions()}</div>}
-            {this.state.executionChecked && <div>{this.getRemoteExecution()}</div>}
-            {!this.state.separateAuth && <div>{this.getCredentials()}</div>}
-          </div>
-          <button onClick={this.handleCopyClicked.bind(this)}>Copy</button>
-        </code>
-        {this.state.separateAuth && this.isAuthenticated() && (
-          <div>
-            The file below contains your auth credentials - place it in your home directory at{" "}
-            <span className="code">~/.bazelrc</span> or place it anywhere you'd like and add{" "}
-            <span className="code">try-import /path/to/your/.bazelrc</span> in your primary{" "}
-            <span className="code">.bazelrc</span> file.
-            <code data-header="~/.bazelrc">
+        {selectedCredential ? (
```

Comment:
this might be the biggest ternary operator i've ever seen

should we do an early exit instead?

something like
if (this.isAuthEnabled() && !this.state.bazelConfigResponse?.credential?.length) {
  return // Looks like your organization doesn't have any API keys
}

return // Actual stuff